core/src/main/java/jenkins/security/ResourceDomainConfiguration.java

```diff
@@ -61,21 +62,22 @@
  * @see ResourceDomainFilter
  * @see ResourceDomainRootAction
  *
- * @since 2.200
+ * @since TODO
```

Comment:
From a Javadoc perspective, this springs into being in 2.203 or whatever the merge target is.

Reply:
```suggestion
 * @since 2.200, unrestricted since TODO
```

to lessen ambiguity